.rubocop.yml

```diff
@@ -13,6 +13,9 @@ LineLength:
 Style/BlockComments:
   Enabled: false
 
+Style/Semicolon:
+  Enabled: false
+
```

Comment:
The `x = 0`, `x += 1` is a wanted part of proving a stack jump

Reply:
Will open an issue.